src/sysen3/papsav.3

```diff
@@ -68,8 +115,8 @@ PROGRAM SYSMSG
 	CALL RITUAL		;ASSURANCE OF PURITY
 
 	MOVEI COUNT, 1		;SET UP COUNT OF MESSAGE SLOTS
-	LSH COUNT, 0		;instruction patched through abstb1 (SYSMLN)
-	movei pt,0              ; instruction patched through abstb1 (SYSMBF)
+	LSH COUNT, SYSMLNG	;instruction patched through abstb1 (SYSMLN)
```

Comment:
The comment doesn't match the symbol.

Reply:
Yeah. Squoze symbols are only 6 chars long. So that last G is irrelevant. I coded my initial version with SYSMLN, which worked fine, but SYSMSG used the 7-character symbol. I could drop the G, or change comment to use the 7-char symbol. Which would you prefer?

Reply:
I guess i’d prefer to update the comment, since ITS (sysjob) uses the 7-char symbol, even though the G will be ignored.

Reply:
Sounds good, then.  Update the comment.